src/serena/config/serena_config.py

```diff
@@ -490,12 +468,7 @@ def from_config_file(cls, generate_if_missing: bool = True) -> "SerenaConfig":
         def get_value_or_default(field_name: str) -> Any:
             return loaded_commented_yaml.get(field_name, get_dataclass_default(SerenaConfig, field_name))
 
-        if is_running_in_docker():
-            instance.gui_log_window_enabled = False  # not supported in Docker
-        else:
-            instance.gui_log_window_enabled = get_value_or_default("gui_log_window_enabled")
-            # in docker the post_init takes care of setting the address to 0.0.0.0, we ignore the user's setting
-            instance.web_dashboard_listen_address = get_value_or_default("web_dashboard_listen_address")
+        instance.gui_log_window_enabled = get_value_or_default("gui_log_window_enabled")
```

Comment:
`instance.web_dashboard_listen_address = get_value_or_default("web_dashboard_listen_address")`

This is missing now